### ISSUE TITLE
Use different heading tags for "Programs and Services" and "New Programs"

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1523]
On the applicant homepage, we show a heading "Programs and Services" and subheadings like "New Programs" or "Continue". But both headings and subheadings use `<h2>` tags, which is bad for screen readers, since they use heading levels to guide the user through the page hierarchy.

We should make the subheadings use `<h3>`s.

![image](https://user-images.githubusercontent.com/77128518/123718991-7c562080-d84e-11eb-94b5-762567f2b298.png)

![image](https://user-images.githubusercontent.com/77128518/123719004-7fe9a780-d84e-11eb-98b5-fb83f879d520.png)

## Original tester's report

> ACTUAL RESULT:
Section "Programs & Services" & subsection "New Programs" are assigned with heading level 2.  There should be nested level heading heirarchy between sections Vs subsections.
> EXPECTED RESULT: Headings should be present on the page with the appropriate nesting/hierarchy (Nested heading levels) . Header should not be used for text appearance, instead heading should be used for structuring the page.
